test/testrunner.py

```diff
@@ -351,7 +356,9 @@ def main():
         return
 
     if specific_test_type in (
-        "sanity",
+        "security_sanity",
```

Comment:
is it fully necessary to remove "sanity" completely? will potentially make merging easier if this isn't a one way change

Reply:
It's not necessary but it might confuse people reading the code later on as to why there are three types of sanity tests. Although I agree about not making this a one way way. I'll add `sanity` back in.

Reply:
Added `sanity` type back in.